File: CS/CalculatedAlias.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;
using System.Configuration;


namespace CalculatedAlias.Module {
    public sealed partial class CalculatedAliasModule : ModuleBase {
        public CalculatedAliasModule() {
            InitializeComponent();
         
            Product.SetProductNameFormat(    
                ConfigurationManager.AppSettings["ProductNameFormat"],    
                ConfigurationManager.AppSettings["ProductNameAlias"]    
                );   
        }
    }
}

File: VB/CalculatedAlias.Module/Module.vb
Imports DevExpress.ExpressApp
Imports System.Reflection
Imports System.Configuration


Namespace CalculatedAlias.Module
    Public NotInheritable Partial Class CalculatedAliasModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()

            Product.SetProductNameFormat(ConfigurationManager.AppSettings("ProductNameFormat"), ConfigurationManager.AppSettings("ProductNameAlias"))
        End Sub
    End Class
End Namespace
